install/bootstrap_livecd_usb.md

```diff
@@ -517,9 +517,9 @@ Now that the configuration is generated, the LiveCD must be populated with the g
 
    ```bash
    linux# echo "
-   CSM_PATH=/var/www/ephemeral/${CSM_RELEASE}
    CSM_RELEASE=${CSM_RELEASE}
-   PITDATA=/var/www/ephemeral
+   PITDATA=$(lsblk -o MOUNTPOINT -nr /dev/disk/by-label/PITDATA)
+   CSM_PATH=${PITDATA}/${CSM_RELEASE}
```

Comment:
This assignment has the same problem as the one in the remote ISO document -- this is not going to use the PITDATA variable assignment from the line above it. But it also suffers from the concern in my comment for the above line -- we want this to be the CSM_PATH value for the booted PIT node, not the current path.

Reply:
I'll rework this to add these into the environment file later.
I'm doing this work in-tandem with a larger overhaul of these files (MTL-1695), and I want to eliminate the duplication of `/var/www/ephemeral`. The docs shouldn't know what that path is perse, that path can be resolved after the system has booted.